gm4_soul_fragments/data/gm4_soul_fragments/functions/soul_fragment/seek_player.mcfunction

```diff
@@ -0,0 +1,16 @@
+#@s = soul fragment with a linked player nearby (@e[type=minecraft:area_effect_cloud,tag=gm4_soul_fragment,distance=..16])
+#from player/search_fragment
+
+#emit particles near linked player
+particle minecraft:soul ~ ~ ~ .05 0 .05 .015 1 force
```

Comment:
Why force these particles if the player is within 16 blocks anyways?

Reply:
I think these ones are the only ones that have to be forced. Otherwise, Soul Fragments would be completely invisible with particles off (not reduced). You're right in every other case though.